modules/calendar/search.php

```diff
@@ -39,11 +39,11 @@ function keywordFilter($arr) {
 </head>
 <body>
   
-  <div class="container-fluid">
-    <div class="row title">
+  <div class="container-fluid" style="margin-left: 20px; margin-right: 20px;">
+    <div class="row">
```

Comment:
Perfect weird having back to calendar as title tab lol. Didn't ever notice though :). Tested and it works great. As per the margin stuff. Looks okay. I think i will have to do something about a generalised theme for EHR so everything aligns on well not a fan of inline css that much as it might override on media queries etc
Nice one.... i am approving this.